net/shorewall-lite/files/vardir

```diff
@@ -0,0 +1,3 @@
+# move state dir out of ram
+VARDIR=/etc/shorewall-lite/state
```

Comment:
Isn't /tmp removed after reboot? So I am missing the config files then. Or am I missing something.